packages/lumx-react/src/components/index.ts

```diff
@@ -92,6 +92,16 @@ enum AspectRatio {
     square = 'square',
 }
 
+/**
+ * Semantic info about the purpose of the component
+ */
+enum Kind {
```

Comment:
Do we have any other scenarios throughout DS where we are using a sort of `type` or `kind` for a component? I am asking in order to keep coherence in the repo. If we are already using `type` rather than `kind`, maybe it would be great to keep that naming convention :)

Reply:
I agree that `Kind` might not be the best name. Something like `InfoLevel` would be better.
We need this enum in the Notification component, the Message component and the Helper component

Reply:
Not sure of what is best.
It's a bit strange to have `kind={InfoLevel.warning}`

For Notification and Message, the enum could be a different one, since they use `success` instead of `valid` for the `InputHelper` component.
This is a semantic difference ^^

Reply:
If this enum is too specific to the `InputHelper` component, we can just put it in the component file and name it `InputHelperKind`